core/domain/activity_jobs_one_off.py

```diff
@@ -375,24 +375,28 @@ def _migrate_user_id(snapshot_model):
             (str, str). Result info, first part is result message, second is
             additional info like IDs.
         """
-        new_user_ids = []
+        # Only commit_cmds of length 2 are processed by this method.
```

Comment:
Perhaps add an assert statement below this line.

Reply:
Done.